src/config/mongoose_config_parser_toml.erl

```diff
@@ -471,30 +472,700 @@ process_module([Mod|_] = Path, Opts) ->
     Rules = parse_list(Path, V),
     [{ip_access, Rules}];
-module_opt([<<"access">>, <<"mod_register">>|_], V) ->
-    [{access, b2a(V)}];
+module_opt([<<"routes">>, <<"mod_revproxy">>|_] = Path, V) ->
+    Routes = parse_list(Path, V),
+    [{routes, Routes}];
+module_opt([<<"versioning">>, <<"mod_roster">>|_], V) ->
+    [{versioning, V}];
+module_opt([<<"store_current_id">>, <<"mod_roster">>|_], V) ->
+    [{store_current_id, V}];
+module_opt([<<"ldap_groupattr">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_groupattr, b2l(V)}];
+module_opt([<<"ldap_groupdesc">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_groupdesc, b2l(V)}];
+module_opt([<<"ldap_userdesc">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_userdesc, b2l(V)}];
+module_opt([<<"ldap_userid">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_userid, b2l(V)}];
+module_opt([<<"ldap_memberattr">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_memberattr, b2l(V)}];
+module_opt([<<"ldap_memberattr_format">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_memberattr_format, b2l(V)}];
+module_opt([<<"ldap_memberattr_format_re">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_memberattr_format_re, b2l(V)}];
+module_opt([<<"ldap_auth_check">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_auth_check, V}];
+module_opt([<<"ldap_user_cache_validity">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_user_cache_validity, V}];
+module_opt([<<"ldap_group_cache_validity">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_group_cache_validity, V}];
+module_opt([<<"ldap_user_cache_size">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_user_cache_size, V}];
+module_opt([<<"ldap_group_cache_size">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_group_cache_size, V}];
+module_opt([<<"ldap_rfilter">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_rfilter, b2l(V)}];
+module_opt([<<"ldap_gfilter">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_gfilter, b2l(V)}];
+module_opt([<<"ldap_ufilter">>, <<"mod_shared_roster_ldap">>|_], V) ->
+    [{ldap_ufilter, b2l(V)}];
+module_opt([<<"buffer_max">>, <<"mod_stream_management">>|_], V) ->
+    [{buffer_max, V}];
+module_opt([<<"ack_freq">>, <<"mod_stream_management">>|_], V) ->
+    [{ack_freq, V}];
+module_opt([<<"resume_timeout">>, <<"mod_stream_management">>|_], V) ->
+    [{resume_timeout, V}];
+module_opt([<<"stale_h">>, <<"mod_stream_management">>|_] = Path, V) ->
+    Stale = parse_section(Path, V),
+    [{stale_h, Stale}];
 module_opt([<<"host">>, <<"mod_vcard">>|_], V) ->
     [{host, b2l(V)}];
-module_opt([<<"ldap_base">>, <<"mod_vcard">>|_], V) ->
+module_opt([<<"search">>, <<"mod_vcard">>|_], V) ->
+    [{search, V}];
+module_opt([<<"matches">>, <<"mod_vcard">>|_], <<"infinity">>) ->
+    [{matches, infinity}];
+module_opt([<<"matches">>, <<"mod_vcard">>|_], V) ->
+    [{matches, V}];
+module_opt([<<"allow_return_all">>, <<"mod_vcard">>|_], V) ->
+    [{allow_return_all, V}];
```

Comment:
I can't find it in the code at all, as well as `search_all_hosts` and `ldap_uids`... they might have been removed, maybe when replacing LDAP implementation with `outgoing_pools`?

Reply:
You are right, also can't find it in places other than tests and docs, will remove them